action.yaml

```diff
@@ -379,56 +379,34 @@ runs:
            --blackduck.url="${{ env.BlackDuck_Url }}" \
            --detect.blackduck.signature.scanner.snippet.matching="NONE" ${{ env.ADDITIONAL_ARGS }}
 
-    # - name: Blackduck scan tracking
-    #   shell: bash
-    #   if: ${{ always() && env.BlackDuck_Project_Name }}
-    #   run: |
-    #     icon="boom"
-    #     [[ "${{ steps.blackduck-scan.outcome }}" == "success" ]] && icon="white_check_mark"
-
-    #     sed -i "s/blackduck_scan=x/blackduck_scan=${icon}/g" $RUNNER_TEMP/icons.properties
-
-    # - name: OWASP Scan
-    #   if: ${{ env.owasp_Project }}
-    #   uses: dependency-check/Dependency-Check_Action@main
-    #   id: owasp
-    #   with:
-    #     project: ${{ env.owasp_Project }}
-    #     path: '.'
-    #     format: 'ALL'
-    #     out: 'reports'
-    #     args: >
-    #       --disableNodeJS \
-    #       --disableAssembly \
-    #       --disableYarnAudit \
-    #       --enableRetired
-
-    # Tagging
-    - name: Prepare tagging
+    - name: Create Tag
       if: ${{ env.Tag_version }}
-      id: prepare-tagging
+      id: git-tag
       shell: bash
       run: |
-        push="true"
-        if [ "${{ env.push_tag_only }}" == "true" ]; then
-          push="origin ${{ env.Tag_version }} -f"
-        fi
+        git pull origin ${{ github.ref_name }} --rebase --autostash --depth=1
```

Comment:
There might be some cases where the rebase will fail due to conflicting changes. Usually, in cases of conflicting changes, Git pauses the process and prompts the user to manually resolve the conflicts. However, in a CI/CD workflow, it's difficult to resolve those conflicts automatically. Our earlier tagging action `EndBug/add-and-commit@v9` also behaved the same way, stopping the process in case of conflicts.
For now, we can proceed with these changes. If such a conflict scenario occurs in the future, then we will work around it.